.github/workflows/ci.yaml

```diff
@@ -150,6 +150,9 @@ jobs:
         repo: ${{ fromJSON(needs.release.outputs.container_repos) }}
 
     steps:
+      - name: Checkout for public key
```

Comment:
We do not want a checkout of the repo here. To only act the provenance based on released things.